app/bin/quasar-describe

```diff
@@ -522,6 +461,9 @@ function printQuasarConfOptions ({ quasarConfOptions }) {
 }
 
 function describe (api) {
+  if (apiParts.docs) {
```

Comment:
"apiParts.docs" ?
Also, make it fail-safe. If it doesn't have the meta.docsUrl, then don't try to open it. The API files with the meta field will only be available in upcoming quasar v1.5.0+ only.